Samples/algorithms/oracle-synthesis/Program.qs

```diff
@@ -0,0 +1,22 @@
+﻿namespace IntegerFactorization {
```

Comment:
I'd suggest a copyright header here, as well as changing the namespace to Microsoft.Quantum.Samples.OracleSynthesis.

Reply:
ups, thanks!